### PR TITLE
Linked the tab bar to components

### PR DESCRIPTION
**Pull request by DeborahIhesiaba**
What type of PR is this? (check all applicable)
[ ] Feature
[ ] Refactor
[ ] Bug Fix
[x] Optimization
[ ] Documentation Update
[ ] Other:

Description of what this Pull Request does
The tab bar is linked to the welcome component and the terminal component. The first tab connects to the welcome component and subsequent tabs is linked to the terminal component.

Related Tickets & Documents
Closes https://github.com/annonymousauthority/P2PCoder/issues/74

QA Instructions, Screenshots, Recordings
tabbar

### CODE REVIEW

File: src/app/page.js [hunk 1]
```diff
@@ -92,6 +94,15 @@ function Home() {
           </div>
           <div className="bg-[#DCDCE5] dark:bg-[#2F2F3A]"></div>
         </div>
+        <div className="ml-24 w-full h-96 bg-violet-500 p-5">
+          {!items || items?.length === 0 ? (
+            <Welcome />
```
Comment (annonymousauthority) [inline src/app/page.js:99]: I can't find this components

File: src/app/page.js [hunk 2]
```diff
@@ -92,6 +94,15 @@ function Home() {
...
+          ) : items[0]?.active && items[0].title === "Welcome" ? (
```
Comment (annonymousauthority) [inline src/app/page.js:100]: This isn't feel right

File: src/app/page.js [hunk 3]
```diff
@@ -92,6 +94,15 @@ function Home() {
...
+            <Welcome />
```
Comment (annonymousauthority) [inline src/app/page.js:101]: Could you turn this back to just `<div>` 
And not a custom component. You can give it a color so we know and tell it's working

File: src/app/page.js [hunk 4]
```diff
@@ -92,6 +94,15 @@ function Home() {
           </div>
           <div className="bg-[#DCDCE5] dark:bg-[#2F2F3A]"></div>
         </div>
+        <div className="ml-24 w-full h-96 bg-violet-500 p-5">
```
Comment (annonymousauthority) [inline src/app/page.js:97]: I don't think your logic is exactly working. Could you look into this again?

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

**Comment by DeborahIhesiaba** [comment]
tabbar

**Review by 7malikk — APPROVED** [review]

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**